tests/alembic_migrations/versions/v_5c841d2c01d1_reject_whitespace_on_meta_vars/test_smoke.py

```diff
@@ -0,0 +1,182 @@
+"""Smoke tests:
+- Upgrade adds check constraint to meta_vars.net_var_name such that they are valid sql identifiers 
+    (this boils down to values inserted into the column should not contain whitespace)
+- Check constraint can still enter valid data
+- Check constraint rejects bad data after migration
+"""
+
+# -*- coding: utf-8 -*-
+import logging
+import pytest
+from alembic import command
+from sqlalchemy import inspect
+from sqlalchemy.exc import IntegrityError
+from psycopg2.errors import CheckViolation
+
+logger = logging.getLogger("tests")
+
+# this is the revision *before* the migration is to be run.
+down_revision = "6cb393f711c3"
+target_revision = "5c841d2c01d1"
+
+columns_to_apply = [
+    "unit",
+    "standard_name",
+    "cell_method",
+    "display_name",
+    "short_name",
+    "long_description",
+]
```

Comment:
There's an argument in favour of defining this independently of the code under test, but you _could_ consider importing it. Also see notes above re. DRY.

Reply:
I think I want this separate, just because the two are the same shouldn't mean the tests automatically work; IE I want the applied columns and the test columns to be separate as they're conceptually different in this instance, even if practically speaking they are the same.